demo/ghostExample/PlaygroundGhostExample.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import cloneDeep from 'lodash/cloneDeep';
+
+import {logger, markupToolbarConfigs} from '../../src';
+import {Playground} from '../Playground';
+
+import {ghostPopupExtension, ghostPopupToolbarItem} from './ghostExtension';
+
+import '../Playground.scss';
+
+logger.setLogger({
+    metrics: console.info,
+    action: (data) => console.info(`Action: ${data.action}`, data),
+    ...console,
+});
+
+const mToolbarConfig = cloneDeep(markupToolbarConfigs.mToolbarConfig);
+
+mToolbarConfig[2].unshift(ghostPopupToolbarItem);
+
+export const PlaygroundGhostExample = React.memo(() => {
+    return (
+        <Playground
+            settingsVisible
+            markupToolbarConfig={mToolbarConfig}
+            markupConfigExtensions={[ghostPopupExtension]}
+            initial={'Ghost example with markup'}
```

Comment:
Let’s provide more information about the example here:
	1.	What does it demonstrate?
	2.	How to work with it?
	3.	Where is the documentation?

```
This is an example of working with a popup in markup editor mode. To test it:

	1.	Switch to markup mode.
	2.	Find the ghost icon in the toolbar.
	3.	Click on it.

Detailed documentation on how to create similar extensions can be found [here](https://github.com/gravity-ui/markdown-editor/blob/main/docs/how-to-add-text-binding-extension-in-markdown.md).
```
The link to the MD file should be on main: `.../blob/main/docs/...`